add img subdir to public,  add hiHat & bassDrum

tmurvv opened the pull request.

tmurvv commented:
So I have uploaded single color svgs for bass-drum and hi-hat into the resources/icons folder. But again I am unsure how to create another pull request or maybe you only use one per user? I find the tutorials teach you how to do the first one, but not how to follow up :) I will keep trying to find a tutorial that can help.

rgavinc commented:
> So I have uploaded single color svgs for bass-drum and hi-hat into the resources/icons folder. But again I am unsure how to create another pull request or maybe you only use one per user? I find the tutorials teach you how to do the first one, but not how to follow up :) I will keep trying to find a tutorial that can help.

All you need to do is commit your new changes to the same branch. Looks like you figured it out :)

tmurvv commented:
yes. I guess how it works is one pr request per branch. Thank you for the help. I also deleted the pngs I added earlier. So all ready for you to be checked over for approval.
